src/core/ctf.h

```diff
@@ -120,16 +120,8 @@ class CTF {
     float               ParticleShiftGivenAzimuth(float azimuth);
     float               WavelengthGivenAccelerationVoltage(float acceleration_voltage);
 
-    inline float xi(float squared_spatial_frequency) {
-        return PIf * wavelength * squared_spatial_frequency * sample_thickness;
-    };
-
-    inline float sinc_xi(float squared_spatial_frequency) {
-        if ( sample_thickness < 0.0001 )
-            return 1.0;
-        if ( squared_spatial_frequency < 0.000001 )
-            return 1.0;
-        return sinf(xi(squared_spatial_frequency)) / xi(squared_spatial_frequency);
+    inline float ThonRingModulator(float squared_spatial_frequency) {
```

Comment:
Sorry to be a pain but all (symmetric) aberrations modulate the Thon Rings, this is just a bad name.

IntegratedFresnelPropagtor, AverageFresnelPropagtor, or if you don't like those then IntegratedDefocusModulation is at least sorta correct.

Reply:
Ok, I'm going with IntegratedDefocusModulation. FresnelPropagator confuses me too much. I also added a comment pointing to McMullan to make sure everybody knows what it is.

I also renamed all the Evaluate to EvaluatePowerspectrum and ApplyCTF to ApplyPowerspectrum, to make it clearer that it returns the square of the CTF and not the CTF itself.

Reply:
Cool, happy about the compromise. If you're curious at some point, I could sketch out an explanation, but it probably isn't important for practical work. Once the checks pass, I'm happy to sign off on this.